44/README.md

```diff
@@ -0,0 +1,140 @@
+1. To submit a patch, a Contributor SHALL create a Platform pull request back to the project.
+1. A Contributor SHALL NOT commit changes directly to the project.
+1. If the Platform implements pull requests as issues, a Contributor MAY directly send a pull request without logging a separate issue.
+1. To discuss a patch, people MAY comment on the Platform pull request, on the commit, or elsewhere.
+1. To accept or reject a patch, a Maintainer SHALL use the Platform interface.
+1. Maintainers SHOULD NOT merge their own patches except in exceptional cases, such as non-responsiveness from other Maintainers for an extended period (more than 1-2 days).
+1. Maintainers SHALL NOT make value judgments on correct patches.
+1. Maintainers SHALL merge correct patches from other Contributors rapidly.
+1. Maintainers MAY merge incorrect patches from other Contributors with the goals of (a) ending fruitless discussions, (b) capturing toxic patches in the historical record, (c) engaging with the Contributor on improving their patch quality.
+1. The user who created an issue SHOULD close the issue after checking the patch is successful.
+1. Any Contributor who has value judgments on a patch SHOULD express these via their own patches.
+1. Maintainers SHOULD close user issues that are left open without action for an uncomfortable period of time.
+
+### 2.5. Branches and Releases
+
+1. The project SHALL have one branch ("master") that always holds the latest in-progress version and SHOULD always build.
+1. The project SHALL NOT use topic branches for any reason. Personal forks MAY use topic branches.
+1. To make a stable release a Maintainer shall tag the repository. Stable releases SHALL always be released from the repository master.
+
+### 2.6. Evolution of Public Contracts
+
+1. All Public Contracts (APIs or protocols) SHALL be documented.
+1. All Public Contracts SHOULD have space for extensibility and experimentation.
+1. A patch that modifies a stable Public Contract SHOULD not break existing applications unless there is overriding consensus on the value of doing this.
+1. A patch that introduces new features SHOULD do so using new names (a new contract).
+1. New contracts SHOULD be marked as "draft" until they are stable and used by real users.
+1. Old contracts SHOULD be deprecated in a systematic fashion by marking them as "deprecated" and replacing them with new contracts as needed.
+1. When sufficient time has passed, old deprecated contracts SHOULD be removed.
+1. Old names SHALL NOT be reused by new contracts.
+1. A new contract marked as "draft" MUST not be changed to "stable" until all the following conditions are met:
+    1. Documentation has been written and finalised.
+    1. Self-tests exercising the functionality are passing on the principle target platform.
+    1. No major changes in the implementation have happened for at least one public release.
+    1. No major changes in the implementation have happened for at least 6 months.
```

Comment:
I don't think that these two points referring to "the implementation" are a good idea.
This section is about the status/maturity of the contract (e.g. API), not about the status/maturity of its implementation(s). In addition, there may be more than a single implementation. 
But to execute tests, it is required that at least one implementation exists. However, this is already implied by the point in line 113.
Therefore, I would drop lines 114 and 115 entirely.

Reply:
What I was thinking about is the overall behaviour - strictly speaking the API signature might not change, but if behaviour changes after a "major implementation change" then it's an incompatible change.

Does the contract definition, as this RFC stands right now, include this? It doesn't seem to me it does, at least not explicitly - maybe it could be clarified?

Reply:
In my understanding, a contract necessarily specified both syntax and semantics/behaviour, at least partially.
If the definition of "contract" does not state this, I agree it should be clarified.